api/api.proto

```diff
@@ -36,6 +38,11 @@ message PremiereRequest {
   string OutUrl = 6;
 }
 
+message EndStreamRequest {
+  uint32 StreamID = 1;
+  string WorkerID = 2;
```

Comment:
Adding another option to discard the vod (thus disabling transcoding, silence detection and vod upload) would be amazing.

Reply:
This sounds great, I will add this as well! :+1: